docs/src/guide/tutorial.jl

```diff
@@ -0,0 +1,298 @@
+                   [1 0 1.];
+                   [1 1 0.]]
+
+## Load HGH pseudopotential for Silicon
+Si = ElementPsp(:Si, psp=load_psp("hgh/lda/Si-q4"))
+
+## Specify type and positions of atoms
+atoms = [Si => [ones(3)/8, -ones(3)/8]]
+
+## 2. Select model and basis
+model = model_LDA(lattice, atoms)
+kgrid = [4, 4, 4]  # k-point grid (Regular Monkhorst-Pack grid)
+Ecut = 7           # kinetic energy cutoff in Hartree
+basis = PlaneWaveBasis(model, Ecut; kgrid=kgrid)
+
+## 3. Run the SCF procedure to obtain the ground state
+scfres = self_consistent_field(basis, tol=1e-8);
+
+# Notice that DFTK by default applies the convergence tolerance `tol`
+# to the energy difference, such that the norm in the density difference
+# is not yet converged to 8 digits.
+#
+# For all quantities DFTK uses
+# [atomic units](https://en.wikipedia.org/wiki/Hartree_atomic_units),
+# where lengths are in Bohr, energies in Hartree and so on.
+# For conversion to other units, DFTK provides conversion factors
+# in the `DFTK.units` module.
+# The convention is that the conversion factors convert *into* atomic units,
+# therefore:
+
+import DFTK.units: eV
+10eV      # 10eV in Hartree
+#-
+1.2 / eV  # 1.2 Hartree in eV
+
+# Computations take place in the unit cell of a lattice defined by a ``3 × 3``
+# matrix (`model.lattice`) with lattice vectors as columns.
+# The reciprocal lattice `model.recip_lattice`,
+# that is the lattice of Fourier coefficients of functions
+# with the periodicity of the lattice) is defined by the matrix
+# ```math
+# B = 2\pi A^{-T} = 2π {A^{T}}^{-1}
+# ```
+# where ``A`` is the unit cell. As a result:
+
+1/2π * model.recip_lattice' * model.lattice
+
+# !!! warning "Row-major versus column-major storage order"
+#     Julia stores matrices as column-major, but other languages
+#     (notably Python and C) use row-major ordering.
+#     Care therefore needs to be taken to properly
+#     transpose the unit cell matrices ``A`` before using it with DFTK.
+#     For some Python libraries (ASE, pymatgen and abipy) DFTK directly
+#     ships conversion tools, see [`load_lattice`](@ref),
+#     [`load_atoms`](@ref). These are explained further
+#     in examples [Integration with ASE](@ref) and
+#     [Creating supercells with pymatgen](@ref).
+
+# Generally DFTK uses reduced coordinates
+# (also called integer or fractional coordinates) for all vectors
+# in reciprocal or real space. This means that vectors are expressed
+# using the columns (lattice unit vectors) of the matrices ``A`` and ``B``,
+# respectively.
+# In the code vectors in real space are denoted by ``r`` if they reside inside
+# the unit cell and by ``R`` for lattice vectors. Vectors in
+# reciprocal space are analogously ``k`` (for vectors in the Brillouin zone)
+# and ``G`` for vectors on the reciprocal lattice.
+# For more details on DFTK's conventions see
+#md # [Notation and conventions](@ref).
+#nb # https://juliamolsim.github.io/DFTK.jl/dev/guide/conventions.html.
```

Comment:
Does this really belong here? I'd just remove it, or just have one half-sentence and a link to the other section